### PR TITLE
made Board page

### LINKED ISSUE #31: Board Page
image
- ignore sun for now
- work directly into board page
- use title and board component
- create a data file (can be dummy) that maps through each member in an array
- /page/board/page.js

### PR DESCRIPTION
**Pull request by AgentEinstein**
image

### CODE REVIEW

File: src/app/board/page.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const Meet = () => {
+  const boardMembers = Array.from({ length: 12 }, () => ({
+    name: "Evan Littlecat",
+    position: "Position Name",
+  }));
+
+  return (
+    <div className="flex flex-col items-center justify-start min-h-screen pt-10">
+      <Title text="Meet the Board" />
+      <div className="grid grid-cols-4 gap-12 mt-10">
```
Comment (seanquiambao) [inline src/app/board/page.js]: y-axis gap looks fine, but we need a little more space horizontally

File: src/app/board/page.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import Board from "@/components/board/Board";
+
+const Meet = () => {
+  const boardMembers = Array.from({ length: 12 }, () => ({
+    name: "Evan Littlecat",
+    position: "Position Name",
+  }));
+
+  return (
+    <div className="flex flex-col items-center justify-start min-h-screen pt-10">
```
Comment (seanquiambao) [inline src/app/board/page.js]: add a little more padding to the top of page

**Review by seanquiambao — CHANGES_REQUESTED** [review]

File: src/app/board/page.js (hunk 1 above)
Comment (AgentEinstein) [reply src/app/board/page.js]: image
Is this better? I added some more horizontal space and increased the top padding slightly

Comment (seanquiambao) [reply src/app/board/page.js]: add a little more spacing

Comment (AgentEinstein) [reply src/app/board/page.js]: horizontally?

Comment (seanquiambao) [reply src/app/board/page.js]: yeah horizontally

Comment (AgentEinstein) [reply src/app/board/page.js]: How about this?
image

Comment (seanquiambao) [reply src/app/board/page.js]: that should be good 👍👍

**Comment by AgentEinstein** [comment]
image

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**